redaxo/src/addons/structure/plugins/content/lib/article_action.php

```diff
@@ -73,7 +73,7 @@ public function exec($type)
             $action = $row->getValue('code');
             $action = str_replace($this->vars['search'], $this->vars['replace'], $action);
             $action = rex_var::parse($action, rex_var::ENV_BACKEND | rex_var::ENV_INPUT, 'action', $this->sql);
-            require rex_stream::factory('action/' . $row->getValue('id') . '/' . $type, $action);
+            rex_stream::include('action/' . $row->getValue('id') . '/' . $type, $action, $this);
```

Comment:
`include` geht nicht als Methodenname in PHP 5, in PHP 7 ginge es.

Reply:
in R5.8 könnten wir es dann also fixen mit dem richtigen methodennamen... wie wichtig ist der fix?

Reply:
ok passt